shell/browser/media/media_stream_devices_controller.cc

```diff
@@ -209,4 +217,28 @@ void MediaStreamDevicesController::HandleUserMediaRequest() {
       std::unique_ptr<content::MediaStreamUI>());
 }
 
+void MediaStreamDevicesController::HandleDisplayMediaRequest() {
+  blink::MediaStreamDevices devices;
+
+  if (request_.video_type ==
+          blink::mojom::MediaStreamType::DISPLAY_VIDEO_CAPTURE &&
+      !request_.requested_video_device_id.empty()) {
+    content::DesktopMediaID video_media_id =
+        content::DesktopMediaID::Parse(request_.requested_video_device_id);
+
+    if (video_media_id.type != content::DesktopMediaID::TYPE_NONE) {
+      devices.emplace_back(blink::mojom::MediaStreamType::DISPLAY_VIDEO_CAPTURE,
+                           video_media_id.ToString(), "Display Video");
+    }
+  }
+
+  VLOG(0) << "DEVICES " << devices.size();
```

Comment:
This can be removed.

Reply:
Removed that whole file change, it was actually an experiment to use `getDisplayMedia` and should not have been committed.